services/app-web/src/components/Select/ProgramSelect/ProgramSelect.stories.tsx

```diff
@@ -1,18 +1,14 @@
 import { Story } from '@storybook/react'
 import { ProgramSelect, ProgramSelectPropType } from './ProgramSelect'
-import { mockMNState } from '../../../testHelpers/apolloMocks'
 
 export default {
     title: 'Components/Select/ProgramSelect',
     component: ProgramSelect,
 }
 
-const statePrograms = mockMNState().programs
-
 export const Default: Story<ProgramSelectPropType> = () => (
     <ProgramSelect
         name="programSelect"
-        statePrograms={statePrograms}
         programIDs={['ea16a6c0-5fc6-4df8-adac-c627e76660ab']}
```

Comment:
This storybook component is broken. The `ProgramSelect` component uses the `useStatePrograms` hook to get state programs. That hook then uses the `useAuth` hook which errors out because that `useAuth` is a can of dependency worms that Storybook does not have.

Not sure exactly what the fix here is, but we essentially have to "mock" the `useStatePrograms` to return some test programs. Maybe spin this work off on its own ticket.

Reply:
FYI @JasonLin0991 no need to despair about dependency worms in storybook for future - you can get those dependencies in storybook - see `ProvidersDecorator`. It works very similar to what we do in tests to work around this.